File: threads/src/main/java/im/threads/internal/transport/threadsGate/responses/BaseResponse.java
package im.threads.internal.transport.threadsGate.responses;

import com.google.gson.JsonObject;

import im.threads.internal.transport.threadsGate.Action;

public class BaseResponse {
    private Action action;
    private String correlationId;
    private JsonObject data;

    public Action getAction() {
        return action;
    }

    public String getCorrelationId() {
        return correlationId;
    }

    public JsonObject getData() {
        return data;
    }
}

File: src/main/java/im/threads/business/transport/threadsGate/responses/BaseResponse.kt
package im.threads.business.transport.threadsGate.responses

import com.google.gson.JsonObject
import im.threads.business.transport.threadsGate.Action

class BaseResponse {
    val action: Action? = null
    val correlationId: String? = null
    val data: JsonObject? = null
}
